install/jakartaee/bin/sig-test_se8.map

```diff
@@ -189,31 +189,31 @@ jakarta.validation=2.0_se8
 
 # Dependency Injection for Java 1.0
 # (This is JSR-330)
-jakarta.inject=1.0_se8
+jakarta.inject=2.0_se8
 
 # Contexts and Dependency Injection (CDI) for Java EE 2.0
 # (This is JSR-365)
 jakarta.enterprise=2.0_se8
 jakarta.decorator=1.1_se8
 
 # Concurrency (JSR 236)
-jakarta.enterprise.concurrent=1.0_se8
+jakarta.enterprise.concurrent=2.0_se8
 
 # JSON-P 1.1
 # (This is JSR-374)
 jakarta.json=2.0.0_se8
 
 # JSON-B
 # (This is JSR-367)
-jakarta.json.bind=1.0_se8
+jakarta.json.bind=2.0_se8
 
 # WebSocket 1.1
 # (This is JSR-356)
```

Comment:
This should be 2.0 shouldn't it?

Reply:
Yes, your right:
```
ag --ignore .git -l "WebSocket 1.1"
internal/docs/jakartaee/JavaEESpecAssertions.xml
install/jakartaee/bin/sig-test_se8.map
install/jakartaee/bin/sig-test_se9.map
install/jakartaee/bin/sig-test_se11.map
install/websocket/docs/README.txt
install/websocket/docs/WebSocketTCK1.1-ReleaseNotes.html
user_guides/jakartaee/src/main/jbake/content/intro.adoc
user_guides/websocket/src/main/jbake/assets/_config.yml
```

Reply:
Just created a [general tracking issue for correcting incorrect version references](https://github.com/eclipse-ee4j/jakartaee-tck/issues/304), although fixes are welcome in any pr for incorrect versions.